Makefile

```diff
@@ -2,7 +2,7 @@ SHELL=/bin/bash
 
 all: net/lp_rpc.pb.go core/test_segment.go livepeer livepeer_cli
 
-net/lp_rpc.pb.go: net/lp_rpc.proto
+net/lp_rpc.pb.go: net/*.proto
```

Comment:
Make targets look at *outputs*, and running protoc on `redeemer.proto` will generate `redeemer.pb.go`, rather than `lp_rpc.pb.go`. Also, using `net/*.proto` is telling Make that all the proto files are dependent in order to generate `lp_rpc.pb.go`, which isn't correct either; it only depends on that one `lp_rpc.proto` file.

Try something like this (untested)

```make
all: net/lp_rpc.pb.go net/redeemer.pb.go core/test_segment.go ...
net/%.pb.go: net/%.proto
    protoc -I=. --go_out=plugins=grpc:. $^
```

(although tbh I'd be fine with just keeping everything in that `lp_rpc.proto` file as well)

Reply:
I added the redeemer and its mocks separately to the makefile, does it check out ?